app/templates/user.html

```diff
@@ -1,25 +1,6 @@
 {% extends "base.html" %}
 {% block css %}
```

Comment:
We can remove these `{% block css %}` and corresponding `{% endblock css %}`

Reply:
I checkout to the branch and found the a lot of styles are disturbed. Please check.

Reply:
Ah.  I renamed the css file and forgot to fix the spelling in the base.html file.  

I have make the changes you suggested and fixed the styles.  They are rendering correctly now.

Reply:
I will resubmit PR.  Thanks